### PR TITLE
I/O methods for seurat3

### PR DESCRIPTION
**Pull request by pcm32**
This PR adds convenience methods to read and write from and to different formats to Seurat, so that these methods can be added to all methods in r-seurat scripts.

These methods have been tested with the newer r-loom conda package merged (https://github.com/bioconda/bioconda-recipes/pull/17834).

I have doubts on whether quit() is the best way to report error, or if I should be using some error raising mechanism.

### CODE REVIEW

File: R/seurat3_io.R [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+write_seurat3_object <- function(seurat_object, format, output_path, verbose = FALSE){
+  if(format == "loom") {
+    as.loom(seurat_object, filename = output_path, verbose = verbose)
+  } else if(format == "seurat" ) {
+    saveRDS(seurat_object, file = output_path)
+  } else if(format == "singlecellexperiment") {
+    saveRDS(as.SingleCellExperiment(seurat_object), file = output_path)
+  } else {
+    cat("Format",format,"for output not recognised, failing now.")
```
Comment (pinin4fjords) [inline R/seurat3_io.R]: You can write to standard error properly like: 

```
write("prints to stderr", stderr())
```

File: DESCRIPTION [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Package: workflowscriptscommon
 Type: Package
 Title: Common Functions for Workflow Script Packages
-Version: 0.0.2
+Version: 0.0.3
```
Comment (pinin4fjords) [inline DESCRIPTION]: We seem to have not updated the version file in previous releases (see the releases), but this should be 0.0.5 now

**Comment by pinin4fjords** [comment]
(I switched the branch this was raised on - assume that was the right thing to do?)

File: DESCRIPTION (hunk 2 above)
Comment (pcm32) [inline DESCRIPTION]:
```suggestion
Version: 0.0.5
```

**Review by pinin4fjords — APPROVED** [review]
LGTM

**Pull request merged**